system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -1029,6 +1030,38 @@ impl pallet_remote_proxy::Config for Runtime {
 	type WeightInfo = weights::pallet_remote_proxy::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const DepositPerItem: Balance = system_para_deposit(1, 0);
+	pub const DepositPerByte: Balance = system_para_deposit(0, 1);
+	pub CodeHashLockupDepositPercent: Perbill = Perbill::from_percent(30);
+}
+
+impl pallet_revive::Config for Runtime {
+	type Time = Timestamp;
+	type Currency = Balances;
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeCall = RuntimeCall;
+	type CallFilter = Nothing;
+	type DepositPerItem = DepositPerItem;
+	type DepositPerByte = DepositPerByte;
+	type WeightPrice = pallet_transaction_payment::Pallet<Self>;
+	type WeightInfo = weights::pallet_revive::WeightInfo<Self>;
+	type ChainExtension = ();
+	type AddressMapper = pallet_revive::AccountId32Mapper<Self>;
+	type RuntimeMemory = ConstU32<{ 128 * 1024 * 1024 }>;
+	type PVFMemory = ConstU32<{ 512 * 1024 * 1024 }>;
```

Comment:
Do you have a link to the SDK where we set this limit on the node side?

Reply:
Yes they are configured via the `HostConfigration`:
```
executorParams: [
    {
      MaxMemoryPages: 8,192
    }
    {
      PvfExecTimeout: [
        Backing
        2,500
      ]
    }
    {
      PvfExecTimeout: [
        Approval
        15,000
      ]
    }
  ]
```

Reply:
Thanks! What about the memory pages for parachain nodes? Is this still set on the CLI or also in state somewhere?

Reply:
Just verified both values: Kusama has `8192` memory pages configured as Basti posed above. The runtime memory pages are [hard coded](https://github.com/paritytech/polkadot-sdk/blob/598feddb893f5ad3923a62e41a2f179b6e10c30c/substrate/client/executor/common/src/wasm_runtime.rs#L30) to `2048`.

```
8192 * 64K = 512M
2048 * 64K = 128M
```